plugins/outputs/cloudwatchlogs/pusher.go

```diff
@@ -22,6 +22,7 @@ import (
 const (
 	reqSizeLimit   = 1024 * 1024
 	reqEventsLimit = 10000
+	elaspedPeriod  = 5 * 24 * time.Hour
```

Comment:
nit: misspelled
```suggestion
	elapsedPeriod  = 5 * 24 * time.Hour
```
 but also this isn't descriptive. What is this used for? Change the name to reflect that.

Reply:
5 days is pretty long to wait before emitting the log since we're using the update timestamp now. How did we decide on 5 days?

Reply:
Would you all be open to a 1-day elapsed time for this log warning message? This is a hard one and I see Yared is trying to strike a balance between raising awareness and avoiding noise. IMO 1 day would be better since it is closer in time to when the impact started.